### DISCUSSION TITLE
Conversations System - [Theoretical] Research

### DISCUSSION THREAD

**Discussion by vladolaru** [pixelgrade/nova-blocks#267] [General]
Since we should always keep an open mind and learn what there is to be learned from others, I am starting this **[theoretical] research** discussion.

I intend this as **a place to gather, discuss, and conclude on:**
- various existing commenting systems
- research articles and papers
- experiments of all kinds (both past and present)
- complementary but applicable research from fields like cognitive or behavioral sciences, journalism, community building, etc.

**Comment by vladolaru** [comment]
CORAL is a project from Voxmedia (https://coralproject.net/) that aims at creating "A better commenting experience".

I don't know about the actual system (it's kind of closed-doors, invite-only), but they published their **extensive research:** https://coralproject.net/research/

There are lots and lots of pieces of information in there. Some are too narrow in focus, while others are spot on with our use-cases. Will update as I have read and thought more about it.

**Comment by georgeolaru** [comment]
A good wrap-up from the **Indieweb** community on comments _via_ webmentions: https://indieweb.org/comments

**Reply by vladolaru** [reply]
The more indieweb we can make accessible to the layperson, the better. I'm all for it.

**Reply by vladolaru** [reply]
And there is a pseudo-official WordPress plugin: https://wordpress.org/plugins/indieweb/

**Comment by vladolaru** [comment]
A ML-approach to encourage people to **be more aware of the impact of their comment upon others:** http://www.perspectiveapi.com/#/home

I like their exploration of making the "classic textarea" a bit more informative.

**Comment by vladolaru** [comment]
An open-source (MIT) **embed commenting system, bloat-free:** https://gitlab.com/commento/commento. I like the fact that they are focused on speed. I also like the way that you can self-host or use their cloud offering (with the option to move to a self-hosted setup later on).

**Comment by vladolaru** [comment]
I am going to gather in here some essays that I believe express a thoughtful (but personal) exploration of what could be better around comments online:

-  Towards a better commenting system http://vatsalmehra.com/essays/better-comments.html
-

**Comment by georgeolaru** [comment]
[Bridgy](https://brid.gy/about) offers _webmention_ support as a service.
> Bridgy periodically checks social networks for responses to your posts and links to your web site and sends them back to your site as webmentions. Bridgy also translates the responses to microformats so that your web site can fetch and parse them.
> 
> Bridgy can also publish posts, comments, likes, and retweets from your web site to social networks. When you ask Bridgy to publish a post, it analyzes its microformat data and publishes it using the social network APIs.

Seen live [here](https://simonhearne.com/2021/layout-shifts-webfonts/#comments).

**Comment by georgeolaru** [comment]
[The Final Word](https://wordpress.org/plugins/the-final-word/) plugin provides a way to mark a selected comment as the one that effectively sums up the conversation from the thread. We do have the "highlight" feature to add more emphasis to certain comments — I see "the final word" as a level on top of the current feature.

![image](https://user-images.githubusercontent.com/1632775/115341492-69b9ce00-a1b1-11eb-948f-b021fcec54e9.png)

Seen live on [WordPress.org blog](https://make.wordpress.org/core/2021/04/18/proposal-treat-floc-as-a-security-concern/).

**Comment by georgeolaru** [comment]
[Quality Assurance in a Structured Collaborative Discussion System](https://publications.rwth-aachen.de/record/670223) via [A Better Web is Coming](https://www.lesswrong.com/posts/HaESgyu2BqDMTpujS/a-better-web-is-coming)
> There's an increasing number of people looking for better ways to communicate on the Web and [Wust](https://publications.rwth-aachen.de/record/670223) marks the general direction things are heading–a more efficient, more bottom-up way of collaborating, with more emphasis on healthy community.

**Reply by vladolaru** [reply]
@georgeolaru Where did you get the paper from?

**Reply by georgeolaru** [reply]
@vladolaru It's from a post on the LessWrong community: https://www.lesswrong.com/posts/HaESgyu2BqDMTpujS/a-better-web-is-coming
